### ISSUE TITLE
timestamp as unix time in seconds results in wrong timezone in chart

### ISSUE THREAD

**Issue by Alexxino7** [node-red/node-red-dashboard#485]
### What are the steps to reproduce?
`[{"id":"5f13547c.7fb11c","type":"tab","label":"Flow 1","disabled":false,"info":""},{"id":"25209ad3.fc3b66","type":"inject","z":"5f13547c.7fb11c","name":"","topic":"","payload":"[]","payloadType":"json","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":707.5000076293945,"y":308.75000190734863,"wires":[["f5329cb6.9b7e3"]]},{"id":"c249d224.f52eb","type":"change","z":"5f13547c.7fb11c","name":"Data Array","rules":[{"t":"set","p":"data","pt":"msg","to":"$.feeds.{\"x\":$floor($toMillis(created_at)/1000), \"y\" : field1}","tot":"jsonata"}],"action":"","property":"","from":"","to":"","reg":false,"x":507.50000762939453,"y":248.75000190734863,"wires":[["2f664f90.ef837","a61cb001.e385c"]]},{"id":"2f664f90.ef837","type":"function","z":"5f13547c.7fb11c","name":"Setup payload","func":"series = [];\ndata = [];\nlabels = [];\n\nseries.push(\"Field1\");\nlabels.push(\"Field1\");\ndata.push(msg.data);\n\nmsg.payload = [{\"series\":series, \"data\":data,\"labels\": labels}];\n\nreturn msg;","outputs":1,"noerr":0,"x":781.2500076293945,"y":238.75000381469727,"wires":[["d9ff005f.59ee1","f5329cb6.9b7e3"]]},{"id":"c1a16059.5b758","type":"function","z":"5f13547c.7fb11c","name":"Dataset","func":"msg = \n\n{\n\"channel\":\n{\n\"id\":\"XXXXXXX\",\"name\":\"temperature sensor\",\n\"description\":\"Temperatura / Prototipo 1 Esp8266 /Sensor /PowerBank / Deepsleep \",\n\"latitude\":\"0.0\",\n\"longitude\":\"0.0\",\n\"field1\":\"Temperatura\",\n\"created_at\":\"2018-09-23T13:26:03Z\",\n\"updated_at\":\"2018-09-29T22:48:56Z\",\n\"last_entry_id\":964\n},\n\n\"feeds\":[\n{\"created_at\":\"2018-10-08T14:25:17Z\",\"entry_id\":955,\"field1\":\"693\"},\n{\"created_at\":\"2018-10-08T14:34:58Z\",\"entry_id\":956,\"field1\":\"687\"},\n{\"created_at\":\"2018-10-08T14:44:36Z\",\"entry_id\":957,\"field1\":\"668\"},\n{\"created_at\":\"2018-10-08T14:54:17Z\",\"entry_id\":958,\"field1\":\"693\"},\n{\"created_at\":\"2018-10-08T15:03:54Z\",\"entry_id\":959,\"field1\":\"725\"},\n{\"created_at\":\"2018-10-08T15:13:35Z\",\"entry_id\":960,\"field1\":\"712\"},\n{\"created_at\":\"2018-10-08T15:23:14Z\",\"entry_id\":961,\"field1\":\"687\"},\n{\"created_at\":\"2018-10-08T15:32:54Z\",\"entry_id\":962,\"field1\":\"643\"},\n{\"created_at\":\"2018-10-08T15:42:33Z\",\"entry_id\":963,\"field1\":\"631\"},\n{\"created_at\":\"2018-10-08T15:52:13Z\",\"entry_id\":964,\"field1\":\"668\"}\n]\n}\nreturn msg;","outputs":1,"noerr":0,"x":357.50000762939453,"y":248.75000190734863,"wires":[["c249d224.f52eb"]]},{"id":"37c7edb2.c9a262","type":"inject","z":"5f13547c.7fb11c","name":"Trigger","topic":"","payload":"","payloadType":"date","repeat":"","crontab":"","once":true,"onceDelay":0.1,"x":217.50000762939453,"y":248.75000190734863,"wires":[["c1a16059.5b758"]]},{"id":"d9ff005f.59ee1","type":"debug","z":"5f13547c.7fb11c","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"payload","targetType":"msg","x":1026.250015258789,"y":236.25000381469727,"wires":[]},{"id":"a61cb001.e385c","type":"debug","z":"5f13547c.7fb11c","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"true","targetType":"full","x":683.75,"y":161.25,"wires":[]},{"id":"f5329cb6.9b7e3","type":"ui_chart","z":"5f13547c.7fb11c","name":"","group":"17f81dce.bee602","order":0,"width":0,"height":0,"label":"mere","chartType":"line","legend":"true","xformat":"HH:mm:ss","interpolate":"step","nodata":"","dot":false,"ymin":"","ymax":"","removeOlder":"1","removeOlderPoints":"","removeOlderUnit":"3600","cutout":0,"useOneColor":false,"colors":["#1f77b4","#aec7e8","#ff7f0e","#2ca02c","#98df8a","#d62728","#ff9896","#9467bd","#c5b0d5"],"useOldStyle":false,"outputs":1,"x":926.2500114440918,"y":321.25000381469727,"wires":[[]]},{"id":"75e713af.f73c5c","type":"inject","z":"5f13547c.7fb11c","name":"","topic":"","payload":"[]","payloadType":"json","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":710.5554962158203,"y":591.1110687255859,"wires":[["a9cebaee.6f9e28"]]},{"id":"17d4b8e3.8d61a7","type":"change","z":"5f13547c.7fb11c","name":"Data Array","rules":[{"t":"set","p":"data","pt":"msg","to":"$.feeds.{\"x\":$toMillis(created_at), \"y\" : field1}","tot":"jsonata"}],"action":"","property":"","from":"","to":"","reg":false,"x":510.5554962158203,"y":531.1110687255859,"wires":[["bc9053bd.8b038","3dacc959.63fd76"]]},{"id":"bc9053bd.8b038","type":"function","z":"5f13547c.7fb11c","name":"Setup payload","func":"series = [];\ndata = [];\nlabels = [];\n\nseries.push(\"Field1\");\nlabels.push(\"Field1\");\ndata.push(msg.data);\n\nmsg.payload = [{\"series\":series, \"data\":data,\"labels\": labels}];\n\nreturn msg;","outputs":1,"noerr":0,"x":784.3054962158203,"y":521.1110706329346,"wires":[["e36eaaaf.774988","a9cebaee.6f9e28"]]},{"id":"c65152da.6586c","type":"function","z":"5f13547c.7fb11c","name":"Dataset","func":"msg = \n\n{\n\"channel\":\n{\n\"id\":\"XXXXXXX\",\"name\":\"temperature sensor\",\n\"description\":\"Temperatura / Prototipo 1 Esp8266 /Sensor /PowerBank / Deepsleep \",\n\"latitude\":\"0.0\",\n\"longitude\":\"0.0\",\n\"field1\":\"Temperatura\",\n\"created_at\":\"2018-09-23T13:26:03Z\",\n\"updated_at\":\"2018-09-29T22:48:56Z\",\n\"last_entry_id\":964\n},\n\n\"feeds\":[\n{\"created_at\":\"2018-10-08T14:25:17Z\",\"entry_id\":955,\"field1\":\"693\"},\n{\"created_at\":\"2018-10-08T14:34:58Z\",\"entry_id\":956,\"field1\":\"687\"},\n{\"created_at\":\"2018-10-08T14:44:36Z\",\"entry_id\":957,\"field1\":\"668\"},\n{\"created_at\":\"2018-10-08T14:54:17Z\",\"entry_id\":958,\"field1\":\"693\"},\n{\"created_at\":\"2018-10-08T15:03:54Z\",\"entry_id\":959,\"field1\":\"725\"},\n{\"created_at\":\"2018-10-08T15:13:35Z\",\"entry_id\":960,\"field1\":\"712\"},\n{\"created_at\":\"2018-10-08T15:23:14Z\",\"entry_id\":961,\"field1\":\"687\"},\n{\"created_at\":\"2018-10-08T15:32:54Z\",\"entry_id\":962,\"field1\":\"643\"},\n{\"created_at\":\"2018-10-08T15:42:33Z\",\"entry_id\":963,\"field1\":\"631\"},\n{\"created_at\":\"2018-10-08T15:52:13Z\",\"entry_id\":964,\"field1\":\"668\"}\n]\n}\nreturn msg;","outputs":1,"noerr":0,"x":360.5554962158203,"y":531.1110687255859,"wires":[["17d4b8e3.8d61a7"]]},{"id":"7292fe11.34f14","type":"inject","z":"5f13547c.7fb11c","name":"Trigger","topic":"","payload":"","payloadType":"date","repeat":"","crontab":"","once":true,"onceDelay":0.1,"x":220.5554962158203,"y":531.1110687255859,"wires":[["c65152da.6586c"]]},{"id":"e36eaaaf.774988","type":"debug","z":"5f13547c.7fb11c","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"payload","targetType":"msg","x":1029.3055038452148,"y":518.6110706329346,"wires":[]},{"id":"3dacc959.63fd76","type":"debug","z":"5f13547c.7fb11c","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"true","targetType":"full","x":686.8054885864258,"y":443.6110668182373,"wires":[]},{"id":"a9cebaee.6f9e28","type":"ui_chart","z":"5f13547c.7fb11c","name":"","group":"17f81dce.bee602","order":0,"width":0,"height":0,"label":"mere","chartType":"line","legend":"true","xformat":"HH:mm:ss","interpolate":"step","nodata":"","dot":false,"ymin":"","ymax":"","removeOlder":"1","removeOlderPoints":"","removeOlderUnit":"3600","cutout":0,"useOneColor":false,"colors":["#1f77b4","#aec7e8","#ff7f0e","#2ca02c","#98df8a","#d62728","#ff9896","#9467bd","#c5b0d5"],"useOldStyle":false,"outputs":1,"x":929.3055000305176,"y":603.6110706329346,"wires":[[]]},{"id":"17f81dce.bee602","type":"ui_group","z":"","name":"Group 1","tab":"6aca32b2.07166c","disp":true,"width":"12","collapse":false},{"id":"6aca32b2.07166c","type":"ui_tab","z":"","name":"Home","icon":"dashboard","disabled":false,"hidden":false}]`
### What happens?
The difference between the 2 subflows is the Jsonata in the change nodes.

One uses unix epoch time in seconds, the other in milliseconds.

The resulting time axis of the flow with timestamp in seconds is in a timezone that makes no sense to me.

![image](https://user-images.githubusercontent.com/4653267/55962205-1b47cf80-5c79-11e9-98f2-bf07bfe79f13.png)
### What do you expect to happen?
I expect the same time range in the X axis, whether the time is seconds or in milliseconds
### Please tell us about your environment:

- [ ] Node-RED-Dashboard version: 2.14.0
- [ ] Node-RED version: 0.20.3
- [ ] node.js version: 10.15.1
- [ ] npm version: 6.4.1
- [ ] Platform/OS: windows 10
- [ ] Browser: chrome/firefox

PS: Sorry for the wall of code. I couldn't figure out how to make the code scroll horizontally.

**Comment by dceejay** [comment]
The chart always expects milliseconds so if you send it seconds you will be compacting that back to somewhere near 1970. If you create seconds you need to multiply by 1000 before feeding the chart.

**Comment by Alexxino7** [comment]
I noticed now that it does say at the end of this part  https://github.com/node-red/node-red-dashboard/blob/master/Charts.md#line-charts that it is supposed to be in milliseconds.

Considering that [1] says that epoch time is the number of seconds, I think you shouldn't call it epoch time, if it's in fact epoch time x 1000.

I've made a pull request with what I consider a clearer formulation.

Didn't mean to come off sounding rude. I appreciate your work.

[1] https://en.wikipedia.org/wiki/Unix_time
